frontend/src/fixtures/courseFixtures.js

```diff
@@ -0,0 +1,37 @@
+const coursesFixtures = {
+  twoCourses:
+    [
+      {
+      "quarter": "20211",
+      "courseId": "CMPSC     8  ",
+      "title": "INTRO TO COMP SCI",
+      "description": "Introduction to computer program development for students with little to no programming experience. Basic programming concepts, variables and expressions, data and control structures, algorithms, debugging, program design, and documentation.",
+      "objLevelCode": "U",
+      "subjectArea": "CMPSC ",
+      "unitsFixed": 4,
+    },
+    {
+      "quarter": "20211",
+      "courseId": "CMPSC     16  ",
```

Comment:
same here. Unless this is supposed to be like that?

Reply:
No!!  Leave it alone!